flow/flow.go

```diff
@@ -0,0 +1,412 @@
+	//
+	// Please note that this changes the user ID on endpoint /userinfo and sub claim of the ID Token. It does not change the
+	// sub claim in the OAuth 2.0 Introspection.
+	//
+	// Per default, ORY Hydra handles this value with its own algorithm. In case you want to set this yourself
+	// you can use this field. Please note that setting this field has no effect if `pairwise` is not configured in
+	// ORY Hydra or the OAuth 2.0 Client does not expect a pairwise identifier (set via `subject_type` key in the client's
+	// configuration).
+	//
+	// Please also be aware that ORY Hydra is unable to properly compute this value during authentication. This implies
+	// that you have to compute this value on every authentication process (probably depending on the client ID or some
+	// other unique value).
+	//
+	// If you fail to compute the proper value, then authentication processes which have id_token_hint set might fail.
+	ForceSubjectIdentifier string `json:"force_subject_identifier" db:"forced_subject_identifier"`
+
+	// Context is an optional object which can hold arbitrary data. The data will be made available when fetching the
+	// consent request under the "context" field. This is useful in scenarios where login and consent endpoints share
+	// data.
+	Context sqlxx.JSONRawMessage `json:"context" db:"context"`
+
+	// If set to true means that the request was already handled. This
+	// can happen on form double-submit or other errors. If this is set
+	// we recommend redirecting the user to `request_url` to re-initiate
+	// the flow.
+	WasHandled bool `json:"-" db:"was_used"`
+
+	Error                *consent.RequestDeniedError `json:"-" db:"error"`
+	LoginAuthenticatedAt sqlxx.NullTime              `json:"-" db:"login_authenticated_at"`
+
+	ConsentChallengeID sqlxx.NullString `json:"consent_challenge_id" db:"consent_challenge_id"`
+	ConsentSkip        bool             `json:"consent_skip" db:"consent_skip"`
+	ConsentVerifier    string           `json:"-" db:"consent_verifier"`
+	ConsentCSRF        string           `json:"-" db:"consent_csrf"`
+
+	CHGrantedScope       sqlxx.StringSlicePipeDelimiter `json:"ch_grant_scope" db:"ch_granted_scope"`
+	CHGrantedAudience    sqlxx.StringSlicePipeDelimiter `json:"ch_grant_access_token_audience" db:"ch_granted_at_audience"`
+	CHRemember           bool                           `json:"ch_remember" db:"ch_remember"`
+	CHRememberFor        int                            `json:"ch_remember_for" db:"ch_remember_for"`
+	CHHandledAt          sqlxx.NullTime                 `json:"ch_handled_at" db:"ch_handled_at"`
+	CHWasHandled         bool                           `json:"-" db:"ch_was_used"`
+	CHError              *consent.RequestDeniedError    `json:"-" db:"ch_error"`
+	CHSessionIDToken     sqlxx.MapStringInterface       `json:"-" db:"ch_session_id_token" faker:"-"`
+	CHSessionAccessToken sqlxx.MapStringInterface       `json:"-" db:"ch_session_access_token" faker:"-"`
```

Comment:
Can you please add code docs for these fields? Additionally, it's a bit confusing if we have `CHWasHandled` for consent, but `WasHandled` for login. I'd suggest to either prefix both or neither

Reply:
Sure. I'll prefix both